### PR TITLE
Coreacuk api get method implementation

### LINKED ISSUE #123: CORE.AC.UK API GET Implementation
### Issue Description

Implementation of the core.ac.uk api will be done.

### Steps

- [x] Implementation
- [x] Unit test

### Deadline for the Issue

8.5.2023

### PR DESCRIPTION
**Pull request by uyduranomer**
This pull request currently needs review to be merged.
In my local all the tests were fine however I'm not sure they were performed accurately.
I was expecting them take longer.
Also, I don't know how to run the tests on github.
Any help to check whether they are correct would be nice.

The merge of this pull request closes #123

### CODE REVIEW

**Comment by hakanaktas0** [comment]
There are some unnecessary changes in the django.yml file and I will be adding all the keys in another PR. Can you omit that file from this PR?

File: practice_app/api/tests.py
[the remark this answers is not in the document]
Comment (uyduranomer) [reply practice_app/api/tests.py:1]: Done 👍
I checked the necessary fileds. Some of the fields may be empty in the external source too, so I didn't check them.

Thanks for the valuable comment :)

**Comment by uyduranomer** [comment]
The requested changes have been done.

**Review by defabdullah — APPROVED** [review]

File: practice_app/api/tests.py
[the remark this answers is not in the document]
Comment (Bera0422) [reply practice_app/api/tests.py:1]: Nice, nevertheless you may test if the keys exist even though the values are empty.

**Review by hakanaktas0 — APPROVED** [review]
I reviewed the code and run the tests. All looks good.

**Pull request merged**